### PR TITLE
adding row of big ownCloud users to home page

### PR DESCRIPTION
**Pull request by jospoortvliet**
@jancborchardt here you go, try two succeeding #562

Added first three logos. Forgive horrible test image, I had overlapping logo issues but that's solved now :dart:

### CODE REVIEW

**Comment by jospoortvliet** [comment]
statistics april 6 2016
Current look.

TODO:
- [ ] decide on the 'other' three logo's.
- [ ] ask & add Amnesty International Germany
- [ ] ask & add Airbus (?)
- [ ] ask & add Migros

**Comment by jancborchardt** [comment]
Make the logos dark instead (half-grey) and don’t change the background. The blue is for the header and footer only. :)

**Comment by jospoortvliet** [comment]
@jancborchardt oh, again you crush me - so much work I put to find out how to do it this way ;-)
statistics april 6 2016
state before making it blue & grey background.

**Comment by jospoortvliet** [comment]
And after following orders:
statistics april 6 2016
I reluctantly and grudgingly admit it looks very nice.

Note that we can't put this live until we've had an OK about ARD and Amnesty.

**Comment by jancborchardt** [comment]
NICE! Just details:

* make the logos a slight bit darker. If it was 50% grey, make it 75% ;)
* do not full-capitalize the »TRUSTED« – only »Trusted«
* reduce the size of the TU Berlin logo a tad so it fits better with the others
* can we add the »Amnesty« text below the logo, similar to [this one](http://www.formialibera.it/wordpress/wp-content/uploads/2014/02/logo-Amnesty.jpg) their name is well-known, their logo not so much

Thanks! :)

File: assets/css/styles.css [hunk 1]
```diff
@@ -51,6 +51,15 @@ div.footer-social-icons img {
 	margin: 0 1em 1em 0;
 }
 
+.promote-buttons a {
+	margin-bottom: 10px;
+	width: 250px;
+}
+
+@media (min-width: 1600px) {
+/* trust row with 6 items */
```
Comment (MorrisJobke) [inline assets/css/styles.css]: ? Is this to do?

Comment (jospoortvliet) [inline assets/css/styles.css]: aw, left-over from earlier stuff.

**Comment by jospoortvliet** [comment]
statistics april 20 2017
Current state. But as discussed we might have to remove ARD (for now).

**Comment by jospoortvliet** [comment]
statistics april 20 2017

**Comment by jospoortvliet** [comment]
Bauhaus it is. @karlitschek if you like it this can go live later this week...

**Comment by jancborchardt** [comment]
Great stuff!

**Comment by karlitschek** [comment]
really awesome guys :+1:

**Comment by MorrisJobke** [comment]
@jospoortvliet Nice :+1:

**Comment by danimo** [comment]
@jospoortvliet the link to bauhaus is broken. Can you fix that in a sep. commit?

**Comment by jospoortvliet** [comment]
@danimo done in a separate commit

**Pull request merged**